R/dbWriteTable.R

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+#' @include PrestoConnection.R
+NULL
+
+random_name <- function(n = 10) {
+  
+}
+
+#' @rdname PrestoConnection-class
+#' @inheritParams DBI::dbWriteTable
+#' @param overwrite a logical specifying whether to overwrite an existing table
+#'   or not. Its default is `FALSE`.
+#' @param append,field.types,temporary,row.names Ignored. Included for
+#'   compatibility with
+#'   generic.
+#' @usage NULL
+.dbWriteTable <- function(
+  conn, name, value,
+  overwrite = FALSE, ...,
+  append = FALSE, field.types = NULL, temporary = FALSE, row.names = NA
+) {
+  stopifnot(
+    length(overwrite) == 1 &&
+      is.logical(overwrite) &&
+      !is.na(overwrite)
+  )
+  stopifnot(is.data.frame(value))
```

Comment:
inherits(value, "data.frame")

Reply:
The function body of [base::is.data.frame()](https://github.com/wch/r-source/blob/trunk/src/library/base/R/dataframe.R#L138) is literally `inherits(x, "data.frame")`. Why is it necessary to replace the function call with its body?

Reply:
Of course it is...R is so weird.